testgen/src/light_block.rs

```diff
@@ -117,13 +109,15 @@ impl LightBlock {
             .next_validators(&validators)
             .time(time);
 
-        let commit = Commit::new(header.clone(), 1);
+        Self::new_default_with_header(header)
+    }
 
+    pub fn new_default_with_header(header: Header) -> Self {
```

Comment:
All the `new_*` constructors to `LightBlock` look like they want to be replaced with a builder API, but perhaps not in this PR.

Reply:
Actually, this requires updating tests in various crates accordingly. I’d suggest to carry this out in a separate PR.